org.mpo.Dayon.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.mpo.Dayon</id>
```

Comment:
is this developed by "Broward Metropolitan Planning Organization (MPO)" ? (that's what mpo.org points to)

Reply:
No, those are the initials of the guy, who started the Dayon! app back in 2008 - Marc Polizzi.

Reply:
so it is not a valid app-id. 

https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Ok, so I use a domain under my control